src/Components/ProductListingPage/FilterSection/SortSection/sortByPrice.js

```diff
@@ -0,0 +1,14 @@
+const sortByPrice = (originalData, operation) => {
+  let priceSortedData = originalData;
+  if (operation === "Newest") {
+    return priceSortedData.sort((a, b) =>
+      a.newestArrival === b.newestArrival ? 0 : a.newestArrival ? -1 : 1
```

Comment:
pure functions are preferred, here you are mutating

Reply:
Thank you for pointing it out. I will work on that issue.